codegen/test/rtc/include/rtc/hiprtc_enable_env.hpp

```diff
@@ -0,0 +1,3 @@
+#include <ck/utility/env.hpp>
```

Comment:
We should really avoid including from the main ck repo as that could require device compilation inadvertently. I think using an ifdef to enable hiprtc will remove the need for this anyways.